### ISSUE TITLE
HookWebpackError: Module build failed

### ISSUE THREAD

**Issue by vargasrueda7** [akveo/nebular#3100]
Hi! 👋 
      
Firstly, thanks for your work on this project! 🙂

Today I used [patch-package](https://github.com/ds300/patch-package) to patch `@nebular/theme@9.0.3` for the project I'm working on.

Here is the diff that solved my problem:

```diff
diff --git a/node_modules/@nebular/theme/components/layout/_layout.component.theme.scss b/node_modules/@nebular/theme/components/layout/_layout.component.theme.scss
index e4a53df..fe5c072 100644
--- a/node_modules/@nebular/theme/components/layout/_layout.component.theme.scss
+++ b/node_modules/@nebular/theme/components/layout/_layout.component.theme.scss
@@ -96,7 +96,7 @@
       }
     }
 
-    @media screen and(min-width: $layout-window-mode-max-width + 150px) {
+    @media screen and (min-width: $layout-window-mode-max-width + 150px) {
       $padding-top: calc(#{nb-theme(layout-window-mode-padding-top)} * 0.5);
 
       @include window-mode($padding-top);
```

<em>This issue body was [partially generated by patch-package](https://github.com/ds300/patch-package/issues/296).</em>

**Comment by CrackerakiUA** [comment]
good solution, but better would be to fix this issue on repo it self and also patch-package not working properly on google cloud :(

**Comment by GuilhermeAbreu** [comment]
Prezado boa tarde.

Isso já subiu para uma hotfix?

Vi que a atividade está aberta a +13 dias!

![image](https://user-images.githubusercontent.com/50218062/189198606-e915e60d-a9a5-4df5-a17a-91c75b4c8178.png)

**Comment by sashaqred** [comment]
Should be fixed in [9.1.0-rc.7](https://www.npmjs.com/package/@nebular/theme/v/9.1.0-rc.7)
